src/native/libs/System.Security.Cryptography.Native/opensslshim.c

```diff
@@ -41,6 +44,15 @@ FOR_ALL_OPENSSL_FUNCTIONS
 #define MAKELIB(v)  SONAME_BASE v
 #endif
 
+#if defined(TARGET_ARM) && defined(TARGET_LINUX)
```

Comment:
All these changes should be under `FEATURE_DISTRO_AGNOSTIC_SSL`.

When we are building against distro-specific OpenSSL, the `time_t` size is fixed. We do not need to be resilient variable `time_t` size.

Reply:
Ah ok. Either way is fine then.